prices.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import csv
+import os
+import time
+import calculations
+
+
+def generate_general_statistic(data, filename):
+    item_id_index = 0
+    store_id_index = 1
+    price_index = 2
```

Comment:
this indices are constants for the project. Maybe let's move them into global scope not to define each time?
```python
ITEM_ID_INDEX = 0
STORE_ID_INDEX = 1
PRICE_INDEX = 2
```
It's fine to have globals as long as they are constant